### PR TITLE
Flutter Connection 2024

### PR DESCRIPTION
**Pull request by ueman**
CFP is still TBA

### CODE REVIEW

File: _conferences/flutter-connection-april-2024.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+name: "Flutter Connection"
+website: [https://www.flutterfriends.dev/](https://flutterconnection.io/)
+location: Paris, France
+online: false
+
+date_start: 2024-04-24
+date_end:   2024-04-24
```
Comment (marandaneto) [inline _conferences/flutter-connection-april-2024.md:8]: You need to add a placeholder for `cfp` I guess otherwise CI is unhappy (just put eg 2024-01-01)

Comment (ueman) [reply _conferences/flutter-connection-april-2024.md:8]: This sites templates is really not that flexible :D

```suggestion
date_end:   2024-04-24

cfp:
  start: 2024-01-01
  end:   2024-01-01
  site: https://flutterconnection.io/

```

File: _conferences/flutter-connection-april-2024.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+name: "Flutter Connection"
+website: [https://www.flutterfriends.dev/](https://flutterconnection.io/)
```
Comment (ueman) [inline _conferences/flutter-connection-april-2024.md]:
```suggestion
website: https://flutterconnection.io/
```

File: _conferences/flutter-connection-april-2024.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+website: https://flutterconnection.io/
+location: Paris, France
+online: false
+
+date_start: 2024-04-24
+date_end:   2024-04-24
+
+cfp:
+  start: 2024-01-01
+  end:   2024-01-01
```
Comment (ueman) [inline _conferences/flutter-connection-april-2024.md]: The CFP for the last year was from January 12 to March 13, 2023, so I'll use that as a placeholder. It seems good enough

```suggestion
  start: 2024-01-12
  end:   2024-03-13
```

**Pull request merged**